### PR TITLE
Move all pages into a nested navigation

### PR DESCRIPTION
**Pull request by stevector**
The pull requests moves all pages within the documentation site into one information architecture.

### CODE REVIEW

**Comment by jazzsequence** [comment]
Community feedback: Add option to collapse the side nav (because someone is going to want it)
Referenced AWS docs: https://docs.aws.amazon.com/signin/latest/userguide/aws_builder_id-apps.html

File: gatsby-ssr.js [hunk 1]
```diff
@@ -1,26 +1,42 @@
-import React from 'react'
+import React from 'react';
```
Comment (stevector) [inline gatsby-ssr.js:1]: I don't think there are any substantive changes to this file in this PR. It is just prettier formatting. I'm tempted to revert this file to make it clearer what this PR is and is not actually changing.

File: source/content/guides/drush/04-drupal-commandline.md [hunk 2]
```diff
@@ -16,6 +16,8 @@ product: [--]
 integration: [drush]
 ---
 
+<!--  @todo, move this page out of this guide and to a "Tutorials" section. -->
```
Comment (stevector) [inline source/content/guides/drush/04-drupal-commandline.md:19]:
```suggestion
<!--  @todo, move this page out of this guide and to a "Tutorials" section. https://github.com/pantheon-systems/documentation/issues/9354  -->
```

File: src/components/omniSidebarNav/index.js [hunk 3]
```diff
@@ -7,15 +7,56 @@ import {
   getOmniSidebarActiveSection,
   turnItemsIntoLinks,
 } from './helpers.js';
+
+// @todo, remove the reporting before merging this PR.
```
Comment (stevector) [inline src/components/omniSidebarNav/index.js:11]: I'm moving much of this reporting code out of this file

File: src/components/omniSidebarNav/submenus/support.js [hunk 4]
```diff
@@ -0,0 +1,122 @@
...
+        'guides/disaster-recovery',
+        'Disaster Recovery Playbook',
+      ),
+      getGuideDirectory('guides/professional-services'),
+      // Todo: relocate this to the support guide
+      simpleLink('/troubleshoot', 'Troubleshooting', [
+        simpleLink('/troubleshoot', 'Overview'),
+        simpleLink('/basic-troubleshooting', 'Basic Troubleshooting'),
+        simpleLink('/modules-plugins-known-issues', 'Known Issues', [
+          // @todo, combine some of these pages.
```
Comment (stevector) [inline src/components/omniSidebarNav/submenus/support.js:21]:
```suggestion
          // @todo, combine some of these pages: https://github.com/pantheon-systems/documentation/issues/9356
```

File: src/components/omniSidebarNav/submenus/support.js [hunk 5]
```diff
@@ -0,0 +1,122 @@
...
+        simpleLink('/performance', 'Overview'),
+        //simpleLink('/debug-slow-performance', 'Performance Troubleshooting', [
+        simpleLink('/debug-slow-performance', 'Debugging Slow Performance'),
+        simpleLink('/debug-cache', 'Debugging Caching'),
+        getGuideDirectory(
+          'guides/new-relic',
+          'New Relic Performance Monitoring',
+        ),
+
+        // Should could move to the education menu.
```
Comment (stevector) [inline src/components/omniSidebarNav/submenus/support.js:56]:
```suggestion
        // Consider moving to the education/tutorials section.
```

File: src/components/omniSidebarNav/submenus/terminus.js [hunk 6]
```diff
@@ -0,0 +1,58 @@
+import { getGuideDirectory, simpleLink } from './../helpers';
+
+const terminus = () => {
+  return {
+    link: '/terminus',
+    title: 'Command Line Interface (CLI)',
+    children: [
+      // @todo, copy the order from the terminus template
```
Comment (stevector) [inline src/components/omniSidebarNav/submenus/terminus.js:8]:
```suggestion
```

File: src/components/omniSidebarNav/submenus/workflows.js [hunk 7]
```diff
@@ -0,0 +1,59 @@
...
+      simpleLink('/develop', 'Overview'),
+      // TODO: Report for Submenus and children lists that do not start with a sublink
+      simpleLink('/pantheon-workflow', 'Deployment Pipeline on Pantheon', [
+        simpleLink('/pantheon-workflow', 'Pantheon Workflow'),
+        simpleLink('/start-state', 'Start States'),
+        getGuideDirectory('guides/multidev'),
+        getGuideDirectory('guides/environment-configuration'),
+        simpleLink('/connection-modes', 'Connection Modes'),
+        getGuideDirectory('guides/sftp', 'SFTP'),
+        //Todo: relocate hotfixes into git guide
```
Comment (stevector) [inline src/components/omniSidebarNav/submenus/workflows.js:17]:
```suggestion
        //Todo: relocate hotfixes into git guide https://github.com/pantheon-systems/documentation/issues/9357
```

**Pull request merged**